test/unit/aws/test_deploy_vm_bucket.py

```diff
@@ -13,6 +13,7 @@
 from test.mock_cast import mock_cast
 
 
+# TODO: Do we need to generalize these tests apply them to Example Data S3 Bucket, too?
```

Comment:
Whom to ask?

Reply:
I aligned with @tomuben and we decided to generalize the test cases. 
See latest push.